packages/pn-pa-webapp/src/pages/NewApiKey.page.tsx

```diff
@@ -46,7 +46,7 @@ const NewApiKey = () => {
   });
 
   const validationSchema = yup.object({
-    name: yup.string().required(tkp('form-error-name')),
+    name: yup.string().min(3, tkp('form-error-minimum')).required(tkp('form-error-name')),
```

Comment:
Why the min set to 3? Is there some specification in the openapi definition? 
If not, replace the `.min(3, tkp('form-error-minimum'))` with `.matches(dataRegex.noSpaceAtEdges, t('no-spaces-at-edges'))`.
If yes, the fix not solves the bug because we can add 3 or more spaces and the error will disappear. In this case we have to add the `.matches(dataRegex.noSpaceAtEdges, t('no-spaces-at-edges'))` in addition to the `.min(3, tkp('form-error-minimum'))`